react/config-overrides.js

```diff
@@ -0,0 +1,25 @@
+const SentryWebpackPlugin = require('@sentry/webpack-plugin');
```

Comment:
could you please remove `react/config-overrides.js` from commit and add it to .gitignore? (this is an artifact of another new change)